tests/framework-tests/core/js/ModelTransformationTests.js

```diff
@@ -48,7 +48,12 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         floaty2: -9876.789,
         hundredInString: "100",
         floatInString: "12.52",
-        floaty2InString: "-9876.789"
+        floatInStringScale1: "12.5",
+        floatInStringScale0: "13",
+        floaty2InString: "-9876.789",
```

Comment:
I'm worried that you haven't tested your original use case - I would put in as one of the test numbers the expression 0.1 + 0.2, and try to display it to, say 2 decimal places and ensure that it appears as 0.30. For a number on the other side, try 0.1 + 0.7